### PR TITLE
speed: Remove the now unused num in the print_message args

### PR DESCRIPTION
**Pull request by rurban**
speed.c: mark and then remove num print_message args
    
these num args went unused with the removal of the ifndef SIGALRM
branches, commit ee1d7f1d25ef24f111f13dc742474cd9c39c2753 Feb 2021
PR #14228

##### Checklist
- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: apps/speed.c [hunk 1]
```diff
@@ -891,7 +887,7 @@ static int EVP_Update_loop_aead(void *args)
     return realcount;
 }
 
-static long rsa_c[RSA_NUM][2];  /* # RSA iteration test */
+//static long rsa_c[RSA_NUM][2];  /* # RSA iteration test */
```
Comment (t8m) [inline apps/speed.c:890]: Forgotten removal.

**Review by t8m — APPROVED** [review]
LGTM. We'll squash the commits when merging.

**Comment by t8m** [comment]
Still OK

**Comment by t8m** [comment]
Please do not rebase and force push unless there is a conflict.

**Review by paulidale — APPROVED** [review]

**Comment by t8m** [comment]
Merged to master branch. Thank you for your contribution.

**Pull request closed without merging**